{{ cookiecutter.project_name }}/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM python:3-slim
 
 EXPOSE 80
+CMD ["./bin/run-prod.sh"]
```

Comment:
Nope! `CMD` defines the default command to run on `docker run`. Since this does not change often we can just move it to the top of the file along with `EXPOSE`.